internal/manifests/collector/rbac.go

```diff
@@ -46,41 +58,49 @@ func ClusterRole(params manifests.Params) *rbacv1.ClusterRole {
 			Labels:      labels,
 		},
 		Rules: rules,
-	}
+	}, nil
 }
 
-func ClusterRoleBinding(params manifests.Params) *rbacv1.ClusterRoleBinding {
-	configFromString, err := adapters.ConfigFromString(params.OtelCol.Spec.Config)
+func ClusterRoleBinding(params manifests.Params) (*rbacv1.ClusterRoleBinding, error) {
+	otelCol, err := convert.V1Alpha1to2(params.OtelCol)
+	if err != nil {
+		return nil, err
+	}
+	confStr, err := yaml.Marshal(&otelCol.Spec.Config)
```

Comment:
thought: I wonder if we should just make this a method on the config object, given we have to do this in a few places

Reply:
y, right now it would be useful. But as soon as `params` contain the `v1alpha2` it becomes irrelevant again.

Reply:
we'll still need to marshal the config regardless to populate the config map and also to generate a hash right?

Reply:
yes